packaging/nuget/NServiceBus.Transport.RabbitMQ.nuspec

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<package xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
-    <id>NServiceBus.RabbitMQ</id>
-    <title>NServiceBus RabbitMQ</title>
-    <version>$version$</version>
-    <authors>$authors$</authors>
-    <owners>$owners$</owners>
-    <licenseUrl>$licenseUrl$</licenseUrl>
-    <projectUrl>$projectUrl$</projectUrl>
-    <iconUrl>$iconUrl$</iconUrl>
-    <requireLicenseAcceptance>$requireLicenseAcceptance$</requireLicenseAcceptance>
-    <description>RabbitMQ support for NServicebus</description>
-    <releaseNotes></releaseNotes>
```

Comment:
Yeah, this shouldn't matter. Maybe if they ever update the gallery to include an actual link, we can add this.